snippets/is_contained_in.md

```diff
@@ -0,0 +1,20 @@
+---
+title: is_contained_in
+tags: list,intermediate
+---
+
+Returns `True` if the elements of the first λιστ are contained in the second one regardless of order, `False` otherwise.
+
+Use `count()` to check if any value in `a` has more occurences than it has in `b`, returing `False` if any such value is found, `True` otherwise.

```

Comment:
```suggestion
Use `count()` to check if any value in `a` has more occurences than it has in `b`, returning `False` if any such value is found, `True` otherwise.

```